var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -530,6 +532,8 @@ def configure_args(self):
             # Suppress the redundant check for m4:
             config_args.append("ac_cv_prog_NC_M4=false")
 
+        config_args.extend(self.enable_or_disable("logging"))
```

Comment:
This is a suboptimal place in the code to put this. Please, move it:
```patch
--- a/var/spack/repos/builtin/packages/netcdf-c/package.py
+++ b/var/spack/repos/builtin/packages/netcdf-c/package.py
@@ -434,6 +434,8 @@ def configure_args(self):
 
         config_args += self.enable_or_disable("fsync")
 
+        config_args += self.enable_or_disable("logging")
+
         if any(self.spec.satisfies(s) for s in ["+mpi", "+parallel-netcdf", "^hdf5+mpi~shared"]):
             config_args.append("CC={0}".format(self.spec["mpi"].mpicc))
 
@@ -532,8 +534,6 @@ def configure_args(self):
             # Suppress the redundant check for m4:
             config_args.append("ac_cv_prog_NC_M4=false")
 
-        config_args.extend(self.enable_or_disable("logging"))
-
         lib_search_dirs.extend(d for libs in extra_libs for d in libs.directories)
         # Remove duplicates and system prefixes:
         lib_search_dirs = filter_system_paths(dedupe(lib_search_dirs))
```

Reply:
done, thanks